__tests__/animals.test.js

```diff
@@ -0,0 +1,79 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('backend-express-template routes', () => {
```

Comment:
don't forget to update your test description - this gets output when you run the tests so it help figure out where things are breaking

Reply:
Good to know! Thanks!